manipulate_pim_data/rule/general_information_on_rule_format.rst

```diff
@@ -282,13 +282,14 @@ Two parameters are required, two other are optional.
 Remove
 ______
 
-This action removes values to a multiselect, a category or a collection.
+This action removes values from a multiselect, a category or a collection.
 
-Two parameters are required, two other are optional.
-- field: attribute code.
-- locale: local code for which value is assigned (optional).
-- scope: channel code for which value is assigned (optional).
-- items: attribute values to remove.
+Two parameters are required, three other are optional.
+ - field: attribute code or "categories".
+ - locale: locale code for which value is assigned (optional).
+ - scope: channel code for which value is assigned (optional).
+ - items: attribute values to remove.
```

Comment:
attribute values ? You mean attribute options ?

Reply:
I went with plain 'values to remove' instead